debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorTask.java

```diff
@@ -209,8 +211,30 @@ public synchronized void start(Configuration config) {
                     throw new ConnectException(
                             "The MySQL server does not appear to be using a row-level binlog, which is required for this connector to work properly. Enable this mode and restart the connector.");
                 }
-                // We're going to start by reading the binlog ...
-                chainedReaderBuilder.addReader(binlogReader);
+
+
+
+                // if there are new tables
+                if (newTablesInConfig()) {
+                    // and we are configured to run a parallel snapshot
+                    if (taskContext.snapshotNewTables() == MySqlConnectorConfig.SnapshotNewTables.PARALLEL) {
+                        ParallelSnapshotReader parallelSnapshotReader = new ParallelSnapshotReader(config, taskContext, getNewFilters(offsets, config));
+
+                        MySqlTaskContext unifiedTaskContext = createAndStartTaskContext(config, getAllFilters(config));
+                        long newTablesBinlogReaderServerId = unifiedTaskContext.serverId() + 30000;
```

Comment:
Where's this 30000 coming from? Somewhere an offset of 10000 is mentioned, but this seems to be something different?